pytorch_lightning/utilities/imports.py

```diff
@@ -85,7 +90,7 @@ def _compare_version(package: str, op, version) -> bool:
 _NEPTUNE_GREATER_EQUAL_0_9 = _NEPTUNE_AVAILABLE and _compare_version("neptune", operator.ge, "0.9.0")
 _OMEGACONF_AVAILABLE = _module_available("omegaconf")
 _POPTORCH_AVAILABLE = _module_available("poptorch")
-_RICH_AVAILABLE = _module_available("rich")
+_RICH_AVAILABLE = _module_available("rich") and _compare_version("rich", operator.ge, "10.2.2")
```

Comment:
Maybe I might have missed the discussion on how we decided on `10.2.2`? What `rich` library change/feature in `v10.2.2` prompted us to do it?

Reply:
@ananthsub mentioned that with earlier versions of Rich this lead to crashes! This might be because rich has been installed a very long time ago and not updated. I may also be wrong but I noticed that earlier versions of rich from 10.2.2 had their own issues.